bpfctl/src/main.rs

```diff
@@ -43,7 +43,21 @@ enum Commands {
     /// Unload a BPF program using the UUID.
     Unload(UnloadArgs),
     /// List all BPF programs loaded via bpfd.
-    List,
+    List(ListArgs),
+    /// Get a program's metadata information specified by kernel id.
+    Get { kernel_id: u32 },
```

Comment:
Add a new issue/follow-up item for both `bpfctl get` and `bpfctl unload` to take either a KernelId or UUID. One parameter, if its a number we assume it to be a KernelId, o.w. assume it's a UUID/

Reply:
Good point, done!